examples/seismic/acoustic/wavesolver.py

```diff
@@ -27,7 +28,10 @@ def __init__(self, model, source, receiver,
         self.source = source
         self.receiver = receiver
 
-        self.time_order = time_order
+        self.time_order = 2
```

Comment:
I don't think this is right, because the value forced here is ultimately the value passed to the operator constructors, so we will always run 2nd order. Do you mean to always set the `time_order=2` for the wavefield symbols `u` and `v`, instead, even if we're running 4th order?

Reply:
>so we will always run 2nd order

For this wave equation this is the case. There is some high-order solvers but these won;t come from "pure FD" but from some multi-level time-stepping schemes.

We are still running a second order in time wave-equation, but it's formulation allows a 4th order discretization error (O(dt^4) but only d/dt^2). It would be risky in my opinion to leave it a free parameter that would lead people to do bad stuff, specially now that I put back `solve`, the time discretization (u.dt2) would break everything (unconditionally unstable for order 4 FD in time).